.github/workflows/gradle-publish-current.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
+
+# The "current" release is built manually from a given git sha and it puts the artifacts
+# that are meant to be used currently on a permalink.
+
+name: Release current artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_git_sha:
+        type: string
+        description: The git sha of the commit from where the current release will be built from.
+        required: true
+
+jobs:
+  call-publish-base:
+    uses: ./.github/workflows/gradle-publish-base.yml
+    with:
+      release_git_sha: '${{ inputs.release_git_sha }}'
+      release_name: 'current'
```

Comment:
See my comment on the internal change but I thing that it should be "release" and not "current"